tests/test_move_embargo_args.py

```diff
@@ -1,39 +1,289 @@
+        for dt in registry_to.queryDatasets(datasetType=..., collections=...)
+    ]
+    print(ids_in_temp_to)
+    # verifying the contents of the temp_to butler
+    # check that what we expect to move (ids_should_be_moved)
+    # are in the temp_to repo (ids_in_temp_to)
+    for ID in ids_should_be_moved:
+        assert ID in ids_in_temp_to, f"{ID} should be in {temp_to} repo but isnt :("
+    # check that all ids currently in the temp_to butler (ids_in_temp_to)
+    # are in what we expect to move (ids_should_be_moved)
+    # this is different from the above because it will trigger if
+    # there is anything in temp_to that wasn't expected to be there
+    # whereas the above will trigger if anything that
+    # should have moved is not in there
+    for ID in ids_in_temp_to:
+        assert ID in ids_should_be_moved, f"{ID} should not be in {temp_to} repo but it is"
+
+    # now check the temp_from butler and see what remains
+    butler_from = Butler(temp_from)
+    registry_from = butler_from.registry
+    ids_in_temp_from = [
+        dt.dataId.full["exposure"]
+        for dt in registry_from.queryDatasets(datasetType=..., collections=...)
+    ]
+    # verifying the contents of the from butler
+    # if move is on, only the ids_remain should be in temp_from butler
+    if move == "True":
+        # checking that everything in temp_from butler
+        # is in the ids_remain list
+        for ID in ids_in_temp_from:
+            assert ID in ids_should_remain_after_move, f"{ID} should not be in {temp_from} repo but it is"
+        # checking that ids_remain are still in the temp_from butler
+        for ID in ids_should_remain_after_move:
+            assert ID in ids_in_temp_from, f"{ID} should not be in {temp_from} repo but it is"
+    # otherwise, if copy
+    else:
+        # everything in temp_from should be either in ids_remain or ids_moved
+        for ID in ids_in_temp_from:
+            assert (ID in ids_should_remain_after_move + ids_should_be_moved), \
+                f"{ID} should be in either {temp_from} or {temp_to} repo but it isn't"
+        # conversely, everything in ids_remain and ids_moved
+        # should be in the temp_from butler
+        for ID in (ids_should_remain_after_move + ids_should_be_moved):
+            assert ID in ids_in_temp_from, f"{ID} should be in {temp_from} repo but it isn't"
+
+
+class TestMoveEmbargoArgs(unittest.TestCase):
+    def setUp(self):
+        temp_dir = tempfile.TemporaryDirectory()
+        temp_from_path = os.path.join(temp_dir.name, "temp_test_from")
+        temp_to_path = os.path.join(temp_dir.name, "temp_test_to")
+        shutil.copytree("data/test_from", temp_from_path)
+        shutil.copytree("data/test_to", temp_to_path)
+        self.temp_dir = temp_dir
+        self.temp_from_path = temp_from_path
+        self.temp_to_path = temp_to_path
+
+        now_time_embargo = "2020-03-01 23:59:59.999999"
+        embargo_hours = 3827088.677299 / 3600  # hours
```

Comment:
This seems a little weird compared to picking a closer `now_time_embargo` and the default 80 hours.

I also think this belongs in the `test()` functions, not `setUp()`.

Reply:
Do you mean we should assign to these outside of setup()? Or do you mean in __name__ == '__main__'? 

We were trying to avoid putting these in every individual test.

Reply:
If i remember correctly, and @jsv1206 can correct this if I'm wrong, the now_time_embargo and the embargo_hours we have here were designed so that observations fall right around the embargo cutoff.

Reply:
`now_time_embargo` moved from `setUp()` to `test()`

Reply:
We are moving the now_time_embargo to the individual tests (no longer in setup). We still have a longer embargo time than 80 hours.

Reply:
@ktlim is your concern with the >80 hour embargo time more of a aesthetic/pedagogical one or related to functionality?

Reply:
It was just aesthetic.